02_Data_Management/Projections_and_Transformations.Rmd

```diff
@@ -150,13 +150,6 @@ plot(publicTransport_CH,las=1,
 
 
 
-### Rescale
```

Comment:
Where is this subchapter now?

Reply:
@ratnanil probably I removed it by mistake. I will add it again